test/playback.test.js

```diff
@@ -28,6 +28,7 @@ QUnit.module('Playback', {
     let done = assert.async();
     let video = document.createElement('video-js');
 
+    // videojs.log.level('debug');
```

Comment:
accident?

Reply:
I think that this code is helpful for debugging, and we should keep the comment there.

Reply:
seems a bit strange to keep a comment in tests though 🤔 It might make sense to break it out as an improvement and have a path to running the tests with debug logging turned on

Reply:
but fine to keep it for now